grate/Migration/FileSystem.cs

```diff
@@ -7,10 +7,14 @@ namespace grate.Migration;
 
 public static class FileSystem
 {
-    public static IEnumerable<FileSystemInfo> GetFiles(DirectoryInfo folderPath, string pattern)
+    public static IEnumerable<FileSystemInfo> GetFiles(DirectoryInfo folderPath, string pattern, bool ignoreDirectoryNames = false)
     {
-        return folderPath
-            .EnumerateFileSystemInfos(pattern, SearchOption.AllDirectories).ToList()
-            .OrderBy(f => Path.GetRelativePath(folderPath.ToString(), f.FullName), StringComparer.CurrentCultureIgnoreCase);
+        return ignoreDirectoryNames 
+            ? folderPath
+                .EnumerateFileSystemInfos(pattern, SearchOption.AllDirectories).ToList()
+                .OrderBy(f => Path.GetFileNameWithoutExtension(f.FullName), StringComparer.CurrentCultureIgnoreCase) 
+            : folderPath
+                .EnumerateFileSystemInfos(pattern, SearchOption.AllDirectories).ToList()
+                .OrderBy(f => Path.GetRelativePath(folderPath.ToString(), f.FullName), StringComparer.CurrentCultureIgnoreCase);
```

Comment:
Not really related, but think this one should perhaps also be updated to ignore the `.sql` extension when ordering scripts and using the subfolders as part of the ordering (default behaviour)

```
.OrderBy(f => Path.GetRelativePath(folderPath.ToString(), Path.GetFileNameWithoutExtension(f.FullName)), StringComparer.CurrentCultureIgnoreCase);
```

This has side effects though.